File: AmpedBiz/AmpedBiz.Service/Dto/UnitOfMeasure.cs
namespace AmpedBiz.Service.Dto
{
    public class UnitOfMeasure
    {
        public virtual string Id { get; set; }

        public virtual string Name { get; set; }

        public UnitOfMeasure() { }

        public UnitOfMeasure(string id, string name)
        {
            this.Id = id;
            this.Name = name;
        }
    }

    public class UnitOfMeasurePageItem
    {
        public virtual string Id { get; set; }

        public virtual string Name { get; set; }
    }
}

File: AmpedBiz/AmpedBiz.Web/src/common/models/unit-of-measure.ts
export interface UnitOfMeasure {
  id?: string;
  name?: string;
  isBaseUnit?: boolean;
  conversionFactor?: number;
  unitOfMeasureClassId?: string;
}

export interface UnitOfMeasurePageItem {
  id?: string;
  name?: string;
  isBaseUnit?: boolean;
  conversionFactor?: number;
  unitOfMeasureClassName?: string;
}